### PR TITLE
Bridge file with upper ontology axioms AND definition normalization/subset addition

### PR DESCRIPTION
**Pull request by turbomam**
create src/bridge directory; remove .gitkeep later

### CODE REVIEW

File: src/ontology/bridge/aio-bridge-to-upper.tsv [hunk 1]
```diff
@@ -0,0 +1,9 @@
+CJM col 1	CJM col 2	AIO current label	AIO propsed label	OBO label	OBO id
+		ID			SC % 
+Bias	disposition	aio:Bias		disposition	BFO:0000016
+Function	rename to MathematicalFunction, place under Planned Process?	aio:Function	aio:MathematicalFunction	planned process	OBI:0000011
+Layer	information artefavt	aio:Layer		information content entity	IAO:0000030
```
Comment (realmarcin) [inline src/ontology/bridge/aio-bridge-to-upper.tsv:5]: typo + different case:

information artefavt

Comment (turbomam) [reply src/ontology/bridge/aio-bridge-to-upper.tsv]: thanks that column has been removed

**Comment by turbomam** [comment]
This PR also normalizes a few more definitions and adds two more subsets to account for knowledge that was removed in the definition normalization process

**Review by caufieldjh — APPROVED** [review]
Looks good! I think we're covered on upper level ontologies now.

**Review by realmarcin — APPROVED** [review]
looks good

**Pull request merged**